.eslintrc

```diff
@@ -30,6 +30,42 @@
       {
         "ignore": ["firebase", "classes", "history", "children"]
       }
+    ],
+    "import/order": [
+      "error",
+      {
+        "newlines-between": "always-and-inside-groups",
```

Comment:
aha I've found the offender

Reply:
I was a little conflicted with this choice, while I like newlines to separate groups of imports, I also don't want to enforce no newlines within the groups themselves so I used this, but looks like it didn't work out too well...

Reply:
how about always?

Reply:
anyhow - I trust that you have more visibility into this than I do so just pick whatever makes sense and we can move on